src/safeposix/syscalls/net_calls.rs

```diff
@@ -1425,9 +1418,13 @@ impl Cage {
                 return listenret;
             }
     
-            let mut garbage_remote = addr.clone();
+            let spin_val = interface::RustRfc::new(interface::RustAtomicBool::new(false));
+            let mut garbage_remote = portlessaddr.clone();
             let thishandle2 = this.clone();
+            
+            let sub_spin_val = spin_val.clone();
             let acceptor = interface::helper_thread(move || {
+                sub_spin_val.store(true, interface::RustAtomicOrdering::Relaxed);
                 let accret = thishandle2.accept_syscall(sock1fd, &mut garbage_remote);
                 if accret < 0 {
                     let sockerrno = match Errno::from_discriminant(unsafe{*libc::__errno_location()} as i32) {
```

Comment:
I know this wasn't updated but weren't we not going to use libc crate in the safeposix code?

Reply:
Yeah I forgot to merge from networking, that's fixed now.